### DISCUSSION TITLE
Need help to setup a predefined path for file browser widget when chosen

### DISCUSSION THREAD

**Discussion by Ms-Ajith** [chriskiehl/Gooey#832] [General]
Hello Guys. I am new to Gooey and I am trying to create a GUI for my work. So for file browser widget, how can I setup a predefined dir to open as default rather to select every time?

**Comment by chriskiehl** [comment]
Checkout the various options available to Gooey in the [docs](https://github.com/chriskiehl/Gooey/blob/master/docs/Gooey-Options.md).  You can pass a `default_path` option when using `DirectoryChooser`. 

Here's an example: 

```python
parser.add_argument("DirectoryChooser", widget="DirChooser",
                        gooey_options={
                            'wildcard':
                                "Comma separated file (*.csv)|*.csv|"
                                "All files (*.*)|*.*",
                            'message': "pick folder",
                            'default_path': r"C:\Users\Chris\Documents\stuff"
                        })
```

**Reply by Ms-Ajith** [reply]
Thanks a lot for the reply. This is working for me now :)
